Makefile

```diff
@@ -73,13 +79,47 @@ dist:
 potfile:
 	$(MAKE) -C po potfile
 
-po-pull:
-	@which zanata > /dev/null 2>&1 || echo "You may not have the Zanata client installed, don't be surprised if the operation fails."
-	zanata pull $(ZANATA_PULL_ARGS)
+# po-pull and update-pot are "inspired" by corresponding Anaconda code at
+# https://github.com/rhinstaller/anaconda/blob/master/Makefile.am
+# Our use case is slightly simpler (only one pot file), but we don't use automake,
+# so there have to be some differences.
 
-push-pot: potfile
-	@which zanata > /dev/null 2>&1 || echo "You may not have the Zanata client installed, don't be surprised if the operation fails."
-	zanata push $(ZANATA_PUSH_ARGS)
+po-pull:
+	TEMP_DIR=$$(mktemp --tmpdir -d oscap-anaconda-addon-l10n-XXXXXXXXXX) && \
+	git clone --depth 1 -b $(GIT_L10N_BRANCH) -- $(L10N_REPOSITORY) $$TEMP_DIR && \
+	cp $$TEMP_DIR/$(OAA_PARENT_BRANCH)/*.po po/ && \
+	rm -rf $$TEMP_DIR
+
+# This algorithm will make these steps:
+# - clone localization repository
+# - copy pot file to this repository
+# - check if pot file is changed (ignore the POT-Creation-Date otherwise it's always changed)
+# - if not changed:
+#   - remove cloned repository
+# - if changed:
+#   - add pot file
+#   - commit pot file
+#   - tell user to verify this file and push to the remote from the temp dir
+POTFILE_BASENAME = oscap-anaconda-addon.pot
+update-pot:
```

Comment:
When am I supposed to run `make update-pot`? Is it run during the release?

Reply:
That's the interesting part. Ideally, that should happen after every PR merge, but it was never done that way even before. So it if fine when we do it upon release.